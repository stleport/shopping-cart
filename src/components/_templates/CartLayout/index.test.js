import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import CartLayout from '.';

function CartLayoutSetup(newProps = {}) {
  const props = {
    children: '',
    footer: ''
  };

  const CartLayoutJSX = (
    <CartLayout {...props} />
  );

  const CartLayoutComponent = shallow(CartLayoutJSX);

  return {
    props,
    newProps,
    CartLayoutJSX,
    CartLayoutComponent,
    // CartLayoutComponentConnect
  };
}

describe('CartLayout', () => {
  it('renders correctly CartLayout component', () => {
    const { CartLayoutJSX } = CartLayoutSetup();
    const CartLayoutComponent = renderer.create(CartLayoutJSX).toJSON();
    expect(CartLayoutComponent).toMatchSnapshot();
  });
});
