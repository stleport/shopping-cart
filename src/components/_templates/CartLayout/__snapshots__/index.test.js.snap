// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartLayout renders correctly CartLayout component 1`] = `
Array [
  <main>
    <div
      className="st-Page__wrapper"
    >
      
    </div>
  </main>,
  <footer
    className="App__footer"
  >
    <a
      className="App__link"
      href="https://frichti.co"
      rel="noopener noreferrer"
      target="_blank"
    >
      frichti.co
    </a>
  </footer>,
]
`;
